pkg/skaffold/schema/latest/v1/config.go

```diff
@@ -524,7 +524,7 @@ type DeployConfig struct {
 // time for hybrid workflows.
 type DeployType struct {
 	// DockerDeploy *alpha* uses the `docker` CLI to create application containers in Docker.
-	DockerDeploy *DockerDeploy `yaml:"-,omitempty"`
+	DockerDeploy *DockerDeploy `yaml:"docker,omitempty"`
```

Comment:
this config just got `released`. 😅

Reply:
this just modifies the `latest/v1/config.go`, so this change should be fine